pkg/kubectl/cmd/patch_test.go

```diff
@@ -53,8 +53,8 @@ func TestPatchObject(t *testing.T) {
 	cmd.Flags().Set("output", "name")
 	cmd.Run(cmd, []string{"services/frontend"})
 
-	// uses the name from the file, not the response
-	if buf.String() != "service/frontend\n" {
+	// uses the name from the response
```

Comment:
@smarterclayton verify we should be printing the server-returned result of applying the patch